extensions/sheetreader/description.yml

```diff
@@ -0,0 +1,75 @@
+extension:
+  name: sheetreader
+  description: Fast XlSX file importer
+  version: 0.1.0
+  language: C++
+  build: cmake
+  excluded_platforms: windows_amd64_rtools
+  license: MIT
+  maintainers:
+    - freddie-freeloader
+
+repo:
+  github: polydbms/sheetreader-duckdb
+  ref: 4c9a97acd678f192d16bd711d93e4883a9ced7bb
+
+docs:
+  hello_world: |
+    -- Create table from XLSX file & use default values for parameters
+    CREATE TABLE data AS FROM sheetreader('data.xlsx');
+
+    -- Example usage of available named parameters
+    CREATE TABLE data2 AS FROM sheetreader(
+      'data2.xlsx',
+      sheet_index=1,
+      threads=16,
+      skip_rows=0,
+      has_header=TRUE,
+      types=[BOOLEAN,VARCHAR],
+      coerce_to_string=TRUE,
+      force_types=TRUE
+    );
+
+
+  extended_description: |
+    ## About SheetReader
+
+    This DuckDB extension, SheetReader, allows you to read XLSX files by using https://github.com/polydbms/sheetreader-core.
```

Comment:
Maybe: Sheetreader-DuckDB is an extension that allows reading XLSX files into DuckDB tables with Sheetreader, our blazingly fast XLSX parser (https://github.com/polydbms/sheetreader-core )